video.c

```diff
@@ -156,8 +161,14 @@ video_init(int window_scale, char *quality)
 									SDL_TEXTUREACCESS_STREAMING,
 									SCREEN_WIDTH, SCREEN_HEIGHT);
 
+	SDL_RWops *rw = SDL_RWFromMem(icondata, sizeof(icondata));
```

Comment:
Use [SDL_RWFromConstMem](https://wiki.libsdl.org/SDL_RWFromConstMem) and the const can stay.

Reply:
I've changed this to use SDL_RWFromConstMem